frontend/test/jest/tests/ProductListItem.test.tsx

```diff
@@ -0,0 +1,38 @@
+// @ts-nocheck
+import { render, screen } from '@testing-library/react';
+import { ProductListItem } from '@components/product-list/sections/FilterableProductsSection/ProductList';
+import * as ProductSearch from '@components/product-list/sections/FilterableProductsSection/useProductSearchHitPricing';
+import * as AppContext from '@ifixit/app';
+import { mockProduct } from 'test/jest/__mocks__/mockProduct';
+
+describe('renders ProductListItem', () => {
+   beforeEach(() => {
+      ProductSearch.useProductSearchHitPricing = jest.fn().mockReturnValue({
+         price: 5,
+         compareAtPrice: 5,
+         isDiscounted: 5,
+         percentage: 5,
+      });
+
+      AppContext.useAppContext = jest
+         .fn()
+         .mockReturnValue({ ifixitOrigin: 'https://www.cominor.com' });
+   });
+
+   afterEach(() => {
+      jest.clearAllMocks();
+   });
+
+   /**
+    * @jest-environment jsdom
+    */
+   it('renders ProductListItem', () => {
```

Comment:
Seems a bit odd to have the `describe` text and the `it` text be the same. Maybe `describe('ProductListItem test)` and `it('matches the ProductListItem snapshot')`?